### ISSUE TITLE
AG website keyword search engine

### ISSUE THREAD

**Issue by nadirB** [benhackbarth/audioguide#8]
As there is an endless list of keywords ;-) , It seems a search field on AG website documentation might be handy...

N.

**Comment by benhackbarth** [comment]
Agreed, the number of keywords is a bit overwhelming. Let me think on it. I may also make a section of the docs with a list of all objects and their keywords..

**Comment by nadirB** [comment]
Yes. Thanks.
